### PR TITLE
feat: Adds default bug & feature request issue templates

### PR DESCRIPTION
**Pull request by darigovresearch**
Would resolve https://github.com/ethereum/remix-project/issues/4691 if merged

### CODE REVIEW

**Review by TELLYWHALE — APPROVED** [review]
Fixed the edges

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (Aniket-Engg) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: replace safari with firefox here

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (Aniket-Engg) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: update `iOS` with `Windows, Linux or MacOS`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (Aniket-Engg) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Remix is not supported for smart phones, can you please remove this section?

**Review by Aniket-Engg — CHANGES_REQUESTED** [review]

**Comment by darigovresearch** [comment]
Thanks for the feedback @Aniket-Engg ! 

Have made adjustments, do you need anything else to merge the pull request?

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**